### DISCUSSION TITLE
Regridding lat/lon from 2D to 1D

### DISCUSSION THREAD

**Discussion by athom152** [pydata/xarray#8916] [Q&A] [answered]
Hi everyone,
I need help with regridding a climate model data using xarray. Currently, the longitude and latitude arrays are in a 2D format, indexed by dimensions "j" and "i". However, I'm used to working with 1D latitude/longitude arrays. I want to reshape the data back to this format without losing its resolution. Any advice on how to do this in xarray would be much appreciated. I've attached a screenshot of the metadata for reference. Thanks!
![Meta_data](https://github.com/pydata/xarray/assets/135888059/22271c4a-a593-4cb7-b259-9fb5fd25944e)
<img width="1215" alt="xarray" src="https://github.com/pydata/xarray/assets/135888059/f82d0775-a169-402b-8589-6616b13cb92a">

**Comment by andersy005** [comment, marked as answer]
@athom152, have you already looked at [xesmf](https://xesmf.readthedocs.io/en/stable/index.html) ? you might find it useful for your use case.
